core/block/import/markdown/blockconverter_test.go

```diff
@@ -89,3 +89,155 @@ func Test_processFiles(t *testing.T) {
 		assert.Len(t, fileBlocks, 0)
 	})
 }
+
+func Test_processTextBlock(t *testing.T) {
```

Comment:
I suggest using real markdown test data here

testing small internal functions leads to fragile tests and don't cover the whole process of import

Reply:
Fixed